vietnamese/kechuyen.md

```diff
@@ -1 +1,2 @@
-﻿Bạn thích nghe [truyện ma](https://www.youtube.com/watch?v=0dfWjEKXpZk)?
\ No newline at end of file
+﻿Bạn thích nghe [truyện ma](https://www.youtube.com/watch?v=0dfWjEKXpZk)?
+Thế còn [truyện cười](https://www.youtube.com/watch?v=3r_-ZLZCxQs)?
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.